README.md

```diff
@@ -27,15 +27,9 @@ This package provides a KeyShot plugin script that creates jobs for AWS Deadline
 
 1. Install deadline and Pyside6
     - e.g. `pip install deadline[gui]`
-2. Copy or link the file `deadline-cloud-for-keyshot/keyshot_script/Submit to AWS Deadline Cloud.py` to the KeyShot scripts folder.
+2. Copy or link the file `deadline-cloud-for-keyshot/keyshot_submitter/Submit to AWS Deadline Cloud.py` to the KeyShot scripts folder.
     - e.g. On Windows `C:/Users/<USER>/Documents/KeyShot/Scripts`
```

Comment:
nit: Can this be updated to reflect both local and system install locations like we are doing down below